File: 0453-minimum-moves-to-equal-array-elements/0453-minimum-moves-to-equal-array-elements.py
class Solution:
    def minMoves(self, nums: List[int]) -> int:
        nums.sort()
        count = 0
        left = 0
        right = len(nums) - 1
        sub = 0
        
        for i in range (len(nums)):
            if left >= right:
                break
            sub = nums[right] - nums[left]
            count += sub
            right-=1
        return  count   

File: 0453-minimum-moves-to-equal-array-elements/0453-minimum-moves-to-equal-array-elements.js
/**
 * @param {number[]} nums
 * @return {number}
 */
function minMoves(nums) {
    nums.sort((a,b) => a - b)
    let left = 0;
    let right = nums.length - 1;
    let count = 0;
    let sub;

    for(let i = 0; i < nums.length; i++){
        if(left >= right){
            break;
        }
        sub = nums[right] - nums[left]
        count += sub
        right--
    }

  return count
};